doc/api/stream.md

```diff
@@ -2321,6 +2438,38 @@ const myDuplex = new Duplex({
 });
 ```
 
+When using pipeline:
+
+```js
+const { Transform } = require('stream');
+const fs = require('fs');
+
+stream.pipeline(
+  fs.createReadStream('object.json')
+    .setEncoding('utf-8'),
+  new Transform({
+    construct(options, callback) {
```

Comment:
```suggestion
    decodeStrings: false, // Accept string input rather than Buffers
    construct(options, callback) {
```

(Otherwise there’s a redundant decode + encode loop here)